packages/gatsby-source-contentful/src/create-schema-customization.ts

```diff
@@ -584,16 +610,13 @@ export const createSchemaCustomization: GatsbyNode["createSchemaCustomization"]
     const makeRichTextLinksResolver =
       (nodeType, entityType) =>
       async (
-        source,
+        source: IRichTextFieldStructure,
         _args,
         context
       ): Promise<Array<IContentfulEntry> | null> => {
-        const links = getRichTextEntityLinks(source, nodeType)[entityType].map(
-          ({ id }) => id
-        )
-
-        const node = context.nodeModel.findRootNodeAncestor(source)
```

Comment:
Should we mention that this is no more possible when inline objects are not tracked?

`context.nodeModel.findRootNodeAncestor`

Reply:
Yeah, just not sure what's good place for it - we can add jsdocs for https://github.com/gatsbyjs/gatsby/pull/38805/files#diff-15724d4352562f842e4eb9df3a864711b1617ea4ebabc56c2aee4e975bca6767R1782

and mention that this function as well as `@fileByRelativePath` and `fromNode: true` argument for couple of directives (`@link`, `@proxy`, etc) - which all use `findRootNodeAncestor ` internally would not work anymore. 

I'll also do separately do some thinking if we could maybe change implementation of `findRootNodeAncestor` to keep it working without tracking (maybe this could work with `AsyncLocalStorage`, but I suspect it wouldn't due to how `graphql-js` execute queries, but it was a while since I've dug into `graphql-js` so maybe my suspicion is wrong), but probably this wouldn't happen anytime soon